pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -233,6 +236,17 @@ func manageKubeAPIServerConfig(client coreclientv1.ConfigMapsGetter, recorder ev
 	if err != nil {
 		return nil, false, err
 	}
+
+	// sanity check that we create a valid config
```

Comment:
@deads2k we might want to exclude the unsupported config overrides here. Wdyt?

Reply:
For those I am fine with non-strict unmarshal. But the observers concern me most because of tons of unstructured logic there.